app/controllers/coach/feedback_controller.rb

```diff
@@ -1,14 +0,0 @@
-class Coach::FeedbackController < Coach::ApplicationController
```

Comment:
@despo does this mean we also removed feedback access for coaches?

Reply:
It does. I think it's something we can clarify and revisit if necessary as we still get names posted and we don't just render feedback that was specifically left for the coach viewing the page.

Reply:
In addition to this the link was never explicitly exposed to coaches (it was part of the organiser menu) so not many people are/were aware and have ever access this.

Reply:
@despo ok, sure